recipes-wpe/thunder/include/webkitbrowser.inc

```diff
@@ -12,6 +12,10 @@ WEBKITBROWSER_DISKCACHE ?= "0"
 WEBKITBROWSER_XHRCACHE ?= "false"
 WEBKITBROWSER_TRANSPARENT ?= "false"
 WEBKITBROWSER_THREADEDPAINTING ?= "2"
+WEBKITBROWSER_RESOLUTION = "1080p"
+
+# Raspberry PI 3 has 4 cores, set to 4 for RPIs
+WEBKITBROWSER_THREADEDPAINTING_rpi ?= "4"
```

Comment:
this means older pi will suffer please document pi3 being minimum needed

Reply:
I guess only RPI0, as RPI1 won't work anymore with ARMv6 to begin with. But sure will add a comment. 

Alternatively is there a way to determine this from the machine features/config?